src/slide-potentiometer-element.ts

```diff
@@ -0,0 +1,250 @@
+import { css, customElement, html, LitElement, property, svg } from 'lit-element';
+import { analog, ElementPin } from './pin';
+
+@customElement('wokwi-slide-potentiometer')
+export class SlidePotentiometerElement extends LitElement {
+  @property() value = 0;
+  @property() minValue = 0;
+  @property() maxValue = 50;
```

Comment:
can we change those two to `min` / `max` and also set the default max value to 100, to match wokwi-potentiometer's behavior?

Reply:
This one I am a bit skeptical about. For mouse and touch having 0-100 is fine, you could even have larger granularity if you whish. But for keyboard interaction, having too many steps makes it difficult to operate the slider. Moving from one end to the other will take a long time. 

On the other hand some programs reserves arrow keys for fine grained adjustment of elements, e.g. when making drawings. 

I initially picked 0-50 because it was a good compromise, not too coarse and not too fine grained

Reply:
I am doing the change now, if we change our minds we can change it back... 😄

Reply:
I set the input element step attribute to 2, to get the same effective number of steps when using the keyboard, while having the range now be from 0-100